README.md

```diff
@@ -70,6 +70,9 @@
 - **[Store todo data in local Storage]**
 - **[Use css to modify the appearance of the list]**
 - **[Use ES6 to modify the javaScript code]**
+- **[Use the mock storage object you have created in Part 1 of this project to mock the localStorage updates.]**
+- **[Mock the HTML elements to test DOM manipulation functions.]**
+- **[grouping your tests using the describe() method.]**
 
 ### Live Site<a name="Live site"></a>
 
```

Comment:
- [ ] Kindly make you add a working link to your Live Demo as the current one shows an `404` error page once clicked. This will make your documentation be more professional.
![Screenshot from 2023-04-13 13-39-54](https://user-images.githubusercontent.com/83042742/231840610-05b9b547-b1b4-4d59-ba9a-96d186c8b37e.png)